api/models/enrollment_tests.py

```diff
@@ -81,11 +83,34 @@ def __str__(self) -> str:
     @property
     def resulting_level(self) -> str:
         """Returns language level of the student based on amount of correct answers."""
-        correct_answers = self.answers.filter(is_correct=True).count()
-        total_answers = self.answers.count()
-        percentage = correct_answers / total_answers * 100
-        closest_percentage = min(
-            LEVEL_BY_PERCENTAGE_OF_CORRECT_ANSWERS, key=lambda x: abs(x - percentage)
-        )
+        return self.calculate_level([answer.id for answer in self.answers.all()])
```

Comment:
self.answers.values_list('id', flat=True)

Reply:
Done. Yes, I'm not that familiar with QuerySet API yet. And damn how docstrings are sparse in Django!